frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -27,7 +27,10 @@ const CardContent = styled('div')(({ theme }) => ({
 
 const HoverButtonsContainer = styled('div')(({ theme }) => ({
     position: 'absolute',
-    right: theme.spacing(2),
+    background: theme.palette.background.paper,
+    padding: theme.spacing(1),
+    top: theme.spacing(1),
+    right: theme.spacing(1),
```

Comment:
Scouting. This is now better aligned and does not overlap weirdly with the underlying text:

<img width="470" height="188" alt="image" src="https://github.com/user-attachments/assets/f584d0ef-b973-4fc4-8774-a74d39bb5dea" />